components/widgets/forest-change/glad-alerts-simple/index.js

```diff
@@ -0,0 +1,180 @@
+    withInd:
+      'There were {count} GLAD alerts reported in {indicator} in {location} between {startDate} and {endDate}, of which {highConfidencePercentage} were {high confidence alerts}.',
+  },
+  metaKey: 'widget_deforestation_graph',
+  large: false,
+  visible: ['dashboard', 'analysis'],
+  colors: 'loss',
+  chartType: 'pieChart',
+  source: 'gadm',
+  dataType: 'glad',
+  categories: ['summary', 'forest-change'],
+  types: ['country', 'geostore', 'wdpa', 'aoi', 'use'],
+  admins: ['adm0', 'adm1', 'adm2'],
+  datasets: [
+    {
+      dataset: POLITICAL_BOUNDARIES_DATASET,
+      layers: [DISPUTED_POLITICAL_BOUNDARIES, POLITICAL_BOUNDARIES],
+      boundary: true,
+    },
+    {
+      dataset: GLAD_DEFORESTATION_ALERTS_DATASET,
+      layers: [GLAD_ALERTS],
+    },
+  ],
+  sortOrder: {
+    summary: 999,
+    forestChange: 999,
+  },
+  pendingKeys: [],
+  refetchKeys: ['forestType', 'landCategory', 'startDate', 'endDate'],
+  settingsConfig: [
+    {
+      key: 'forestType',
+      label: 'Forest Type',
+      type: 'select',
+      placeholder: 'All tree cover',
+      clearable: true,
+    },
+    {
+      key: 'landCategory',
+      label: 'Land Category',
+      type: 'select',
+      placeholder: 'All categories',
+      clearable: true,
+      border: true,
+    },
+    {
+      key: 'dateRange',
+      label: 'Range',
+      endKey: 'endDate',
+      startKey: 'startDate',
+      type: 'datepicker',
+    },
+  ],
+  settingsBtnConfig: {
+    text: '+ Select an intersection',
+    shouldShowButton: (props) =>
+      !props.settings.forestType && !props.settings.landCategory,
+  },
+  // where should we see this widget
+  whitelists: {
+    adm0: tropicalIsos,
+  },
+  // initial settings
+  settings: {
+    dataset: 'glad',
+  },
+  getData: (params) => {
+    const { GLAD } = params.GFW_META.datasets;
+    const defaultStartDate = GLAD?.defaultStartDate;
+    const defaultEndDate = GLAD?.defaultEndDate;
+    const startDate = params?.startDate || defaultStartDate;
+    const endDate = params?.endDate || defaultEndDate;
+    if (shouldQueryPrecomputedTables(params)) {
+      return fetchGladAlertsSum({
+        ...params,
+        startDate,
+        endDate,
+      }).then((alerts) => {
+        const gladsData = alerts && alerts.data.data;
+        let data = {};
+        if (gladsData && GLAD) {
+          data = {
+            alerts: gladsData,
+            settings: {
+              startDate,
+              endDate,
+            },
+            options: {
+              minDate: '2015-01-01',
+              maxDate: defaultEndDate,
+            },
+          };
+        }
+        return data;
+      });
+    }
+    return fetchAnalysisEndpoint({
+      ...params,
+      params,
+      name: 'glad-alerts',
+      slug: 'glad-alerts',
+      version: 'v1',
+      aggregate: true,
+      aggregateBy: 'days',
+    }).then((alertsResponse) => {
+      const alerts = alertsResponse.data.data.attributes.value;
```

Comment:
use optional chaining to avoid crashes if api is down / properties change: `alertsResponse?.data?.data?.attributes?.value` for safty